.npmignore

```diff
@@ -1,8 +1,14 @@
-.DS_Store
```

Comment:
`.DS_Store` is removed because it is ignored by default:
https://docs.npmjs.com/misc/developers

```
By default, the following paths and files are ignored, so there's no need to add them to .npmignore explicitly:

    .*.swp
    ._*
    .DS_Store
    .git
    .hg
    .npmrc
    .lock-wscript
    .svn
    .wafpickle-*
    config.gypi
    CVS
    npm-debug.log
```